dipy/denoise/noise_estimate.py

```diff
@@ -292,6 +331,6 @@ def estimate_sigma(arr, disable_background_masking=False):
     for i in range(sigma.size):
         convolve(arr[..., i], k, output=conv_out)
         mean_block = np.sqrt(6/7) * (arr[..., i] - 1/6 * conv_out)
-        sigma[i] = np.sqrt(np.mean(mean_block[mask]**2))
+        sigma[i] = np.sqrt(np.mean(mean_block[mask]**2) / factor)
```

Comment:
Wasn't there a note saying you can keep the priority of operation grouped together like in

a**2 + b**2

Plus I really find it counterintuitive to have the exponent not close to the actual number, like in

a ** 2 / b ** 2
Seems messy :/